### PR TITLE
[ROAD-826] fix: Remove manually included dependencies

### PR DESCRIPTION
**Pull request by asaf92**
### Description

Removed manually included DLLs from the solution

### CODE REVIEW

File: Snyk.VisualStudio.Extension/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="snyk_visual_studio_plugin.154948da-e3ba-45ce-9bee-9688a7e30462" Version="1.1.13" Language="en-US" Publisher="Snyk" />
+        <Identity Id="snyk_visual_studio_plugin.154948da-e3ba-45ce-9bee-9688a7e30462" Version="1.1.15" Language="en-US" Publisher="Snyk" />
```
Comment (aldanchenko) [inline Snyk.VisualStudio.Extension/source.extension.vsixmanifest:4]: I saw Michel set this to `0.0.0` [here](https://github.com/snyk/snyk-visual-studio-plugin/pull/164). Just for information

**Comment by aldanchenko** [comment]
And one additional question: did you test this changes on all VS versions (2015-2022)?

File: Snyk.Common/Snyk.Common.csproj [hunk 2]
```diff
@@ -56,8 +56,8 @@
     <Reference Include="Microsoft.Extensions.Primitives, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.Extensions.Primitives.2.0.0\lib\netstandard2.0\Microsoft.Extensions.Primitives.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.13.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```
[the remark this answers is not in the document]
Comment (asaf92) [reply Snyk.Common/Snyk.Common.csproj:59]: Yes, I want to target the lowest version possible which is supported and included in VS2015.
Check this for more info:
https://devblogs.microsoft.com/visualstudio/using-newtonsoft-json-in-a-visual-studio-extension/

File: Snyk.VisualStudio.Extension/source.extension.vsixmanifest (hunk 1 above)
Comment (asaf92) [reply Snyk.VisualStudio.Extension/source.extension.vsixmanifest:4]: > And one additional question: did you test this changes on all VS versions (2015-2022)?

Yes, but I also want to double check with you if there's any case where you think that including the DLLs is necessary.

**Review by aldanchenko — DISMISSED** [review]
LGTM

**Review by michelkaporin — APPROVED** [review]

**Pull request merged**